Tools/Pipeline/Gtk/Global.cs

```diff
@@ -32,10 +34,22 @@ public static void Initalize()
                 DesktopEnvironment = line;
             }
             #endif
+
+            UseHeaderBar = Global.GtkMajorVersion >= 3 && Global.GtkMinorVersion >= 12 && Global.DesktopEnvironment == "GNOME";
```

Comment:
@tomspilman With tis line I check if headerbar shoul bne used. It checks if Gtk is 3.12 or higher and if it;s running GNOME as desktop enviorment.